privacyidea/api/lib/prepolicy.py

```diff
@@ -679,6 +679,7 @@ def check_max_token_user(request=None, action=None):
     Pre Policy
     This checks the maximum token per user policy.
     Check ACTION.MAXTOKENUSER
+    Check ACTION.MAXACTIVETOKENUSER
```

Comment:
I think we also need to add a check to ``enable_token``? Otherwise, users can circumvent the "max active tokens per user" limit by disabling their tokens, enrolling new tokens, and then enabling all of their tokens.

Reply:
You are right. Will do.

Reply:
done in 8d037b56